components/vf-link-list/vf-link-list--easy.njk

```diff
@@ -0,0 +1,25 @@
+<div class="vf-links {%- if component_modifier %} {{ component_modifier }}{% endif %}">
+  <h3 class="vf-links__heading">{{ title }}</h3>
+  <ul class="vf-links__list {%- if list_modifier %} vf-links__list--{{ list_modifier}}{% endif %} | vf-list">
+    {% for item in list %}
+    <li class="vf-list__item">
+      <a class="vf-list__link" href="{{ item.link_list_href }}">
+        {% if item.image %}
+          {% if item.image == "blank" %}
+            <span><!-- no image --></span>
+            {% else %}
+            {{ item.image }}
+          {% endif %}
+        {% endif %}
+        {{ item.text }}
+
+        <svg class="vf-icon vf-icon__arrown--down | vf-list__icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><title>arrow-button-down</title><path d="M.249,7.207,11.233,19.678h0a1.066,1.066,0,0,0,1.539,0L23.751,7.207a.987.987,0,0,0-.107-1.414l-1.85-1.557a1.028,1.028,0,0,0-1.438.111L12.191,13.8a.25.25,0,0,1-.379,0L3.644,4.346A1.021,1.021,0,0,0,2.948,4a1,1,0,0,0-.741.238L.356,5.793A.988.988,0,0,0,0,6.478.978.978,0,0,0,.249,7.207Z"/></svg>
+      </a>
+      {% if item.badge %}{% render '@vf-badge' %}{% endif %}
```

Comment:
Tried using item.bade and item.meta, but it came out weird? Is this meant to be used? 

![image](https://user-images.githubusercontent.com/928100/75864389-3166fd00-5e02-11ea-87b7-30cf662aa1bd.png)

Reply:
Hmmm, not in the current schema of things - but I can see that it might be a good thing. Especially if it's directing to some content on a page that is 'in beta' -- but then wouldn't the whole page be 'in beta' because of that, negating the need for an individual label?

I'll work on seeing what can be done - the problem here is that the meta and badge are 'outside' the `<a>` where the SVG is inside so that it matches state styles.

Reply:
I could see it's use cases ... just seemed off and noted it wasn't in the example, thought maybe it was in-process code that got left behind. Little additional text things could really be nice. Probably a docs thing in the end.

Reply:
![Screenshot 2020-03-04 at 09 48 54](https://user-images.githubusercontent.com/925197/75866771-78062880-5dfd-11ea-824d-51d9a2264d0a.png)

We could do this with a little CSS grid, it would place the SVG outside of the `<a>` tag however - which incurs additional CSS for the states 

The idea is to keep the SVG next to the title, so in this example short text links will have a massive gap before the arrow.

Reply:
![Screenshot 2020-03-04 at 09 52 29](https://user-images.githubusercontent.com/925197/75867038-e21ecd80-5dfd-11ea-873f-bf69c721ba91.png)

Which we could turn to something like this


But does it make sense with the arrow to have the potential badge and meta underneath still?

Reply:
I wonder if the badge is helpful? I mean it should go next to the content that is alpha/beta, right?

I do like the text snippets, but do we _have_ to have it right now? Suppose it could wind up becoming mini paragraphs quite easily, which might be good, might be terrible.

Reply:
> It could wind up becoming mini paragraphs quite easily.

Yeah, and possibly duplicate the content it's trying to direct the user too. 

Perhaps, for now it's a 'do not use it like this' thing